addons/clones/userscript.js

```diff
@@ -1,7 +1,7 @@
 export default async function ({ addon, console, msg }) {
   const vm = addon.tab.traps.vm;
 
-  let showOnlyInEditor = addon.settings.get("editoronly");
+  let alsoShowOnProjectPage = addon.settings.get("editoronly");
```

Comment:
I think you forgot to change the setting id here.
```suggestion
  let alsoShowOnProjectPage = addon.settings.get("projectpage");
```

Reply:
Never mind, that was already fixed.